src/main/scala/com/exasol/cloudetl/bucket/Bucket.scala

```diff
@@ -64,8 +70,50 @@ abstract class Bucket extends LazyLogging {
    *
    * This method also globifies the bucket path if it contains regex.
    */
-  final def getPaths(): Seq[Path] =
-    FileSystemUtil.globWithPattern(bucketPath, fileSystem)
+  final def getPaths(): Seq[Path] = properties.getFileFormat() match {
+    case FileFormat.DELTA => getPathsFromDeltaLog()
+    case _                => FileSystemUtil.globWithPattern(bucketPath, fileSystem)
+  }
+
+  private[this] def getPathsFromDeltaLog(): Seq[Path] = {
+    val spark = createSparkSession()
+    val strippedBucketPath = stripTrailingStar(bucketPath)
+    val deltaLog = DeltaLog.forTable(spark, strippedBucketPath)
+    if (!deltaLog.isValid()) {
+      throw new IllegalArgumentException(
+        s"The provided path: '$bucketPath' is not a Delta format!"
```

Comment:
How can a path be a Delta format? For end users it's a file, isn't it?

Reply:
No, it is path to a folder: `s3a://bucket/data/delta/`. This path should have special directory `_delta_log` that indicates it is Delta format.
Maybe, the error can be more clear. I will update it to `.. is not Delta formatted directory!"`.